recipes/protobuf/all/conanfile.py

```diff
@@ -17,8 +17,8 @@ class ProtobufConan(ConanFile):
     generators = "cmake"
     short_paths = True
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "with_zlib": [True, False], "fPIC": [True, False], "lite": [True, False]}
-    default_options = {"with_zlib": False, "shared": False, "fPIC": True, "lite": False}
+    options = {"shared": [True, False], "with_zlib": [True, False], "fPIC": [True, False], "lite": [True, False], "with_rtti": [True, False]}
+    default_options = {"with_zlib": False, "shared": False, "fPIC": True, "lite": False, "with_rtti": True}
```

Comment:
```suggestion
    options = {
        "shared": [True, False],
        "fPIC": [True, False],
        "with_zlib": [True, False],
        "lite": [True, False],
        "with_rtti": [True, False]
    }
    default_options = {
        "shared": False,
        "fPIC": True,
        "with_zlib": False,
        "lite": False,
        "with_rtti": True
    }
```

Note that I also change the order, so options and default_options match (otherwise it's quite difficult to read).